awx/ui/client/src/inventories-hosts/inventories/related/hosts/related-host.list.js

```diff
@@ -64,7 +64,17 @@ export default ['i18n', function(i18n) {
                 label: i18n._('Hosts'),
                 uiSref: ".edit({inventory_id: host.inventory_id,host_id: host.id})",
                 ngClass: "{ 'host-disabled-label': !host.enabled }",
-                columnClass: 'col-lg-5 col-md-4 col-sm-8 col-xs-7',
+                columnClass: 'col-lg-3 col-md-3 col-sm-8 col-xs-7',
+                dataHostId: "{{ host.id }}",
+                dataType: "host",
+                class: 'InventoryManage-breakWord'
+            },
+            description: {
+                key: true,
+                label: i18n._('Description'),
+                uiSref: ".edit({inventory_id: host.inventory_id,host_id: host.id})",
```

Comment:
See above - this shouldn't be a link.  You can get rid of `key`, `uiSref`, `ngClass`, `dataHostId`, `dataType`, and `class`.